### DISCUSSION TITLE
@babel/eslint-parser is not picking up babel config in monorepo

### DISCUSSION THREAD

**Discussion by ianwalter** [babel/babel#13962] [Q&A] [answered]
I'm trying to use ESLint in a monorepo root, but I'm getting the following error:

```
/__w/swag/swag/packages/swag-sheet/components/ActionMenu.js
8
  18:4  error  Parsing error: This experimental syntax requires enabling one of the following parser plugin(s): 'jsx, flow, typescript' (18:4)
```

I also have `requireConfigFile` set to false in the `parserOptions`, otherwise I would have errors complaining that there is no babel config in the monorepo root (where it is not needed). The babel config for the package above is in it's package.json and works fine, but it seems like it's not being picked up when linting files in the package from root. Basically I want to have a single ESLint config in the monorepo root for all packages and be able to have custom (or no) Babel configurations in each package. This worked with `babel-eslint` but does not seem to work for `@babel/eslint-parser`.

Anyone able to point out if I'm doing something incorrectly or have any other suggestions?

**Comment by nicolo-ribaudo** [comment, marked as answer]
`babel-eslint` completely ignored your config files. You can try creating a top-level `babel.config.json` to tell Babel that you want the `.babelrc` files in `packages` to be loaded:
```json
{
  "babelrcRoots": ["./packages/*"]
}
```
docs: https://babeljs.io/docs/en/options#babelrcroots

If you prefer, you can set it in your eslint config:
```json
{
  "praserOptions": {
    "babelOptions": { "babelrcRoots": ["./packages/*"] }
  }
}
```

**Reply by ianwalter** [reply]
I see. Thank you Nicolo.
